style.css

```diff
@@ -0,0 +1,253 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+html,
+body {
+  font-family: 'Poppins';
+  font-size: 14px;
+  font-weight: normal;
+}
+
+
+header > div {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  border-bottom: 1px solid lightgray;
+  
+ 
+}
+
+nav {
+  width: 600px;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  list-style: none;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
```

Comment:
In your CSS, different types of colour declarations (human language, RGB and Hexadecimal) are used. Try to stick to one.

Reply:
Got it, thanks.